client/src/modules/users/users.js

```diff
@@ -7,7 +7,7 @@ UsersController.$inject = ['$state', 'UserService', 'NotifyService', 'ModalServi
  * Users Controller
  * This module is responsible for handling the CRUD operation on the user
  */
-function UsersController($state, Users, Notify, Modal, uiGridConstants) {
+function UsersController($state, Users, Notify, Modal, DropdownBehavior, uiGridConstants) {
```

Comment:
It doesn't look like you use `DropdownBehavior` anymore.  Go ahead and remove it.